src/Host/Broker/Impl/Resources.resx

```diff
@@ -237,4 +237,7 @@
   <data name="Trace_UserProfileDirectory" xml:space="preserve">
     <value>User {0} profile directory: {1}</value>
   </data>
+  <data name="Exception_MaxAllowedUsers" xml:space="preserve">
+    <value>Broker has reached maximum allowed concurrent user limit.</value>
```

Comment:
If this is user-visible I'd split it into 2 specific messages. `Windows Workstation only allows a singe user session' and 'R Remote Services computer has reached maximum number of concurrent users'